.github/workflows/check-versions.yaml

```diff
@@ -2,7 +2,7 @@ name: Check versions
 
 on:
   schedule:
-    - cron: '0 8 * * 1'
+    - cron: '0 8 * * *'
```

Comment:
Do we really want to increase the interval? Once a week seems sufficient to me.

Reply:
I'll revert it because weekly is still better then random, which we are doing now, and there is contention.

but, how do we know we have cve's without someone looking? This running is the thing that looks and lets us know. So if its weekly, we're only looking for cve fixes weekly. I'd rather know sooner.

The resources for the job are quite minimal, so its not a very high cost.